rset.1

```diff
@@ -47,6 +47,12 @@ accepts one or more hostnames which match the labels in
 .Pp
 The arguments are as follows:
 .Bl -tag -width Ds
+.It Fl A
+Download files listed in label export paths the local
+.Pa _archive
+directory in the format
+.Sq hostname basename(filename) .
```

Comment:
Do you think, spaces in the files names is a good idea?..

Reply:
Spaces are the separator in the label export list, so I figured they are safe to use. Granted, this is maybe surprising